lib/run.js

```diff
@@ -48,16 +50,19 @@ function run (opts, cb) {
   let clients = []
   for (let i = 0; i < opts.connections; i++) {
     let client = new Client(url)
-    client.on('response', record)
+    client.on('response', record(client))
     client.on('connError', onError)
     clients.push(client)
   }
 
-  function record (statusCode, resBytes, responseTime) {
-    statusCodes[(parseInt(statusCode) / 100) - 1] += 1
-    latencies.record(responseTime)
-    bytes += resBytes
-    counter++
+  function record (client) {
+    return function record (statusCode, resBytes, responseTime) {
```

Comment:
I think you can have two different names for this, `buildRecord` and `record` maybe.